src/werkzeug/test.py

```diff
@@ -634,7 +619,7 @@ def server_port(self):
         pieces = self.host.split(":", 1)
         if len(pieces) == 2 and pieces[1].isdigit():
             return int(pieces[1])
-        elif self.url_scheme == "https":
+        if self.url_scheme == "https":
```

Comment:
This (and an earlier place) is a non functional change of an `elif` into an `if`. Did a style checker tell you to do this?

Reply:
I think flake8 might be calling these out. Change seems fine.